semantickernel-experimental/src/main/java/com/microsoft/semantickernel/connectors/data/azureaisearch/AzureAISearchVectorStoreRecordCollection.java

```diff
@@ -104,6 +116,10 @@ public AzureAISearchVectorStoreRecordCollection(
         nonVectorFields.addAll(this.recordDefinition.getDataFields().stream()
             .map(VectorStoreRecordDataField::getEffectiveStorageName)
             .collect(Collectors.toList()));
+
+        storageNames = recordDefinition.getStorageNames();
+        firstVectorFieldName = recordDefinition.getVectorFields().isEmpty() ? null
+            : recordDefinition.getVectorFields().get(0).getName();
```

Comment:
Are you sure getVectorFields() will never return an empty list?

Reply:
It is checked in the line: `recordDefinition.getVectorFields().isEmpty() ? null : recordDefinition.getVectorFields().get(0).getName();`